README.md

```diff
@@ -1,152 +1,164 @@
-packages you need. Fury distribution offers three types of packages:
-
-- **Bases** : Sets of Kustomize bases to deploy necessary components in Kubernetes
-- **Modules**: Terraform modules to deploy Kubernetes infrastructure, and it’s dependencies
-- **Roles**: Ansible roles for deploying, configuring, and managing a Kubernetes infrastructure
-
-In addition to the package manager feature, it enables you to self-provision Fury Clusters.
-Read more about this feature on its documentation site.
-
-## Furyfile
+<h1 align="center">
+  <img src="docs/assets/fury-epta-white.png" width="200px"/><br/>
+  Furyctl
+</h1>
 
-Furyfile is a simple YAML formatted file where you list which packages(and versions) you want to have.
-You can omit a type if you don't need any of its packages. An example Furyfile with packages listed
-would be like the following:
+<p align="center">The CLI tool for the Kubernetes Fury Distribution.</p>
 
-```yaml
-# all sections are optional
-
-# map of prefixes and versions used to force a specific version for all the matching roles/modules/bases
-versions:
-  # e.g. will force version v1.15.4 if the name matches "aws*"
-  aws: v1.15.4
-  monitoring: master
+[![Build Status](https://ci.sighup.io/api/badges/sighupio/furyctl/status.svg)](https://ci.sighup.io/sighupio/furyctl)
+![Release](https://img.shields.io/github/v/release/sighupio/furyctl?label=Furyctl)
+![Slack](https://img.shields.io/badge/slack-@kubernetes/fury-yellow.svg?logo=slack)
+![License](https://img.shields.io/github/license/sighupio/furyctl)
 
-roles:
-  - name: aws/etcd
-  - name: aws/kube-control-plane
-
-modules:
-  - name: aws/aws-vpc
-  - name: aws/aws-kubernetes
+# Furyctl
 
-bases:
-  - name: monitoring
-  - name: logging
-  # versions can be overridden if needed by specifying them for each package
-    version: master
-```
+Furyctl is a simple CLI tool to:
 
-You can get all packages in a group by using a group name *(like `logging`)* or single packages under a group
-(like `monitoring/prometheus-operator`).
+- download and manage the Kubernetes Fury Distribution modules
+- self-provision Fury clusters on AWS, GCP and vSphere
 
 ## Install
 
-### Github Releases
-
 You can find `furyctl` binaries on the [Releases page](https://github.com/sighupio/furyctl/releases).
 
-Supported architectures are *(64 bit)*:
-- `linux`
-- `darwin`
-
-Download the right binary for your architecture, then add it to your `PATH`. Assuming it's downloaded in your
-`~/Downloads` folder, you can run following commands (replacing `{arch}` with your architecture):
+To download the latest `0.6.1` release, run:
 
 ```bash
-chmod +x ~/Downloads/furyctl-{arch}-amd64 && mv ~/Downloads/furyctl-{arch}-amd64 /usr/local/bin/furyctl
+wget -q https://github.com/sighupio/furyctl/releases/download/v0.6.1/furyctl-$(uname -s)-amd64 -O /tmp/furyctl
+chmod +x /tmp/furyctl
+sudo mv /tmp/furyctl /usr/local/bin/furyctl
```

Comment:
This is mac only, but we release linux binaries also :)

Reply:
This would work for both I think.

`furyctl-$(uname -s)-amd64` resolves to `furyctl-darwin-amd64` on MAC OSX and `furyctl-linux-amd64` on Linux